src/main/java/org/apache/commons/io/input/BoundedInputStream.java

```diff
@@ -405,9 +399,9 @@ public synchronized long getCount() {
     }
 
     /**
-     * Gets the max count of bytes to read.
+     * Returns the maximum number of bytes this stream is allowed to read.
      *
-     * @return The max count of bytes to read.
+     * @return the maximum number of bytes permitted, or {@value IOUtils#EOF} if unbounded.
```

Comment:
Please do not change the Commons convention that a sentence starts with a capital letter.